shared/aux.go

```diff
@@ -325,3 +326,59 @@ func fileExists(files []os.DirEntry, workload string) bool {
 
 	return false
 }
+
+func IPv6IPEncloser(ip string)string {
```

Comment:
if the func using this it will be in  the same file no need to export

Reply:
not same file but may be same package, can check and update accordingly